tools/sgcloudsqlproxy/tool.go

```diff
@@ -30,7 +30,7 @@ func PrepareCommand(ctx context.Context) error {
 		runtime.GOARCH,
 	)
 	binURL := fmt.Sprintf(
-		"https://storage.googleapis.com/cloudsql-proxy/v%s/%s",
+		"https://storage.googleapis.com/cloud-sql-connectors/cloudsql-proxy/v%s/%s",
```

Comment:
```suggestion
		"https://storage.googleapis.com/cloud-sql-connectors/cloud-sql-proxy/v%s/%s",
```

filename above should change `_` to `-` also.

Reply:
aha, thanks a lot, let me update :)